CHANGELOG.md

```diff
@@ -6,10 +6,36 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Breaking Changes
+- parameter_key: `lr` parameter has been renamed to `dt` to better reflect the meaning of the parameter, which is the time step of the simulation.
+- solve(): The `solve` method has been removed from the solvers and a `__call__` method has been implemented; users must call the solver object directly.
 
-- Added new feature X
-- Fixed bug Y
-- Improved performance of Z
+### Added
+- LangevinSolver has been added to the package. This solver is a stochastic solver based on the Langevin equation.
+- Updated all solvers with an implementation of the ADAM algorithm. This algorithm can be invoked by calling the solver with the `solve_type` keyword set to `ADAM` and the `hyperparameters` keyword set to a dictionary of hyperparameter names and values.
```

Comment:
@MehmetC-1QBit I have a couple questions that I hope you can help me answer!
How does the user know what parameter names they can give, is it documented somewhere? What are the parameter names?

Reply:
`solve_type` is a control keyword with default `None`. If  it is deliberately set to `'Adam'` then `_solve_adam()` method will be executed with  relevant`hyperparameters`. Otherwise, its default value (i.e. `solve_type=None`) refers to the call for original solve method. The Adam method consist of three `hyperparameters` in which  $\beta_1$ and $\beta_2$ are exponential decay rates for the moment estimates and $\alpha$ is the step size. Please refer to [the paper](https://doi.org/10.48550/arXiv.1412.6980) for more information about Adam algorithm and its hyperparameters.